doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -2353,3 +2353,48 @@ Response code:
             }
           }
         ]
+
+
+### Notify [POST /v2/op/notify{?options}]
+
+This operation is intended to consume a notification payload so that all the entity data included by such notification is persisted, overwriting if necessary.
+This operation is useful when one NGSIv2 endpoint is subscribed to another NGSIv2 endpoint (federation scenarios). 
+The request payload must be an NGSIv2 notification payload. 
+The behaviour must be exactly the same as `POST /v2/op/update` with `actionType` equals to `APPEND`. 
```

Comment:
Typo: equals => equal

Reply:
In addition `APPEND` -> `append`

(Related with https://github.com/telefonicaid/fiware-orion/pull/3197)

Reply:
Fixed in a5de938